### ISSUE TITLE
crashing on alpine linux , libnative-platform-file-events.so

### ISSUE THREAD

**Issue by mio-19** [gradle/gradle#26759]
### Current Behavior

Run gradle under a normal directory and it crashed 

### Expected Behavior

-

### Steps to Reproduce

+ ./gradlew ...

```
Starting a Gradle Daemon (subsequent builds will be faster)                                                                                                             FAILURE: Build failed with an exception.                                                                        * What went wrong:                                      Unable to start the daemon process.                     This problem might be caused by incorrect configuration of the daemon.                                          For example, an unrecognized jvm option is used.        Please refer to the User Manual chapter on the daemon at https://docs.gradle.org/8.0.1/userguide/gradle_daemon.html                                                     Process command line: /usr/lib/jvm/java-17-openjdk/bin/java --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.lang.invoke=ALL-UNNAMED --add-opens=java.prefs/java.util.prefs=ALL-UNNAMED --add-opens=java.base/java.nio.charset=ALL-UNNAMED --add-opens=java.base/java.net=ALL-UNNAMED --add-opens=java.base/java.util.concurrent.atomic=ALL-UNNAMED -Xms1G -Xmx3G -Dfile.encoding=UTF-8 -Duser.country=US -Duser.language=en -Duser.variant -cp /home/p/.gradle/wrapper/dists/gradle-8.0.1-bin/7rsvvmdp4bsd5fhm7a9cyrnzz/gradle-8.0.1/lib/gradle-launcher-8.0.1.jar org.gradle.launcher.daemon.bootstrap.GradleDaemon 8.0.1   Please read the following process output to find out more:                                                      -----------------------                                 #                                                       # A fatal error has been detected by the Java Runtime Environment:                                              #                                                       #  SIGSEGV (0xb) at pc=0x000000000002b1c0, pid=17055, tid=17057                                                 #                                                       # JRE version: OpenJDK Runtime Environment (17.0.8+7) (build 17.0.8+7-alpine-r2)                                # Java VM: OpenJDK 64-Bit Server VM (17.0.8+7-alpine-r2, mixed mode, sharing, tiered, compressed oops, compressed class ptrs, g1 gc, linux-aarch64)                     # Problematic frame:                                    # C  [libnative-platform-file-events.so+0x2eb8c]  _init+0x39e4                                                  #                                                       # No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again                             #                                                       # An error report file with more information is saved as:                                                       # /home/p/.gradle/daemon/8.0.1/hs_err_pid17055.log      #                                                       # If you would like to submit a bug report, please visit:                                                       #   https://gitlab.alpinelinux.org/alpine/aports/issues # The crash happened outside the Java Virtual Machine in native code.                                           # See problematic frame for where to report the bug.    #                                                                                                                                                                       * Try:                                                  > Run with --stacktrace option to get the stack trace.  > Run with --info or --debug option to get more log output.                                                     > Run with --scan to get full insights.                                                                         * Get more help at https://help.gradle.org
```

### Gradle version

8.0.1

**Comment by ov7a** [comment]
Thank you for your interest in Gradle!

This issue will be closed as a duplicate of

* #24875 

Please add your use case and 👍 to that issue.
